lib/token-assert.js

```diff
@@ -230,6 +230,7 @@ TokenAssert.prototype._setEmptyLineIndentation = function(lineNumber, newWhitesp
  *
  * @param {Object} options.token
  * @param {Object} options.nextToken
+ * @param {boolean} [options.moveWhitespaceToNextToken]
```

Comment:
i like the idea of using a callback. I updated the PR to do that instead!

re: different line-wrapping options: yeah, the test deliberately tickles that question. My general philosophy is that if the user put a line break there, we should assume there's a good reason for there to be a line-break in there somewhere.

You could imagine a future jscs rule like "minimizeLineBreaksInExpressions" that warned you if a line-break in an expression was unnecessary, and autofixed it. This would make more sense to me as an orthogonal rule, rather than having the individual operator rules make this decision.